modules/core/public/assets/js/loginuserpass.js

```diff
@@ -1,9 +1,20 @@
 ready(function () {
     var button = document.getElementById("submit_button");
-    button.onclick = function () {
-        this.innerHTML = button.getAttribute("data-processing");
-        this.disabled = true;
+    var form = document.getElementById("f");
+
+    window.onpageshow  = function () {
+        var button = document.getElementById("submit_button");
+        button.innerHTML = button.getAttribute('data-default');
+        button.disabled = false;
+    }
 
+    form.onsubmit = function () {
+        var button = document.getElementById("submit_button");
+        button.innerHTML = button.getAttribute("data-processing");
+        button.disabled = true;
+    }
+
+    button.onclick = function () {
         var form = document.getElementById("f");
         form.submit();
         return true;
```

Comment:
I think this can now be removed? Since the onclick event just replicates what the normal button click would do. Then also line 2 can be removed.